### DISCUSSION TITLE
This version of the Android Support plugin for IntelliJ IDEA (or Android Studio) cannot open this project

### DISCUSSION THREAD

**Discussion by DevPatel8873** [FIRST-Tech-Challenge/FtcRobotController#377] [Q&A] [answered]
This version of the Android Support plugin for IntelliJ IDEA (or Android Studio) cannot open this project, please retry with version 2021.1.1 or newer.

I've tried changing the Gradle versions and Android Gradle Plugin versions however nothing is removing this error. I'm assuming I need to change the SDK. Does anyone know the correct version to set these to in order to start programming?

**Comment by cmacfarl** [comment, marked as answer]
FIRST provides no official support for Android, or FTC SDK, development using Intellij IDEA.  If you are using Intellij IDEA and encountering issues please switch to Android Studio.

The v7.2+ versions of the SDK require Android Studio Chipmunk 2021.2.1 or later.   Please confirm that you are using the latest Android Studio and if you are still having problems, please provide screenshots of the errors.

**Reply by DevPatel8873** [reply]
I am using Android Studio Arctic Fox 2020.3.1. Can you provide a download link for Android Studio chipmunk 2021.2.1?

**Reply by texasdiaz** [reply]
Windows Installer for Chipmunk Patch 2:
https://redirector.gvt1.com/edgedl/android/studio/install/2021.2.1.16/android-studio-2021.2.1.16-windows.exe

You can also download it from the Android Studio archives here (Look at August 3):
https://developer.android.com/studio/archive

**Reply by texasdiaz** [reply]
You can also just download the latest version, Android Studio Dolphin.
https://developer.android.com/studio

**Comment by tom-ricci** [comment]
I had this same issue before--it was [fixed](https://youtrack.jetbrains.com/issue/IDEA-287437) in Ultimate 2022.2, so if you're not up-to-date try that. I'm not sure if it's in the community edition if you're using that though. It took nearly a year for the patch to roll out on ultimate so if you're on community and its not supported its probably a better idea to switch to Android Studio instead of trying to get JetBrains to add it to community.

**Reply by ItsHarper** [reply]
The Android plugin version is always in sync between Ultimate and Community.
